packages/back-end/src/integrations/Athena.ts

```diff
@@ -5,6 +5,8 @@ import { AthenaConnectionParams } from "../../types/integrations/athena";
 import { FormatDialect } from "../util/sql";
 
 export default class Athena extends SqlIntegration {
+  // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+  // @ts-ignore
```

Comment:
I've added `"strictPropertyInitialization": true` to the TSConfig so we can get some help from TypeScript with regards to initializing member variables in the constructor. This property is enabled by default in `strict: true` but we don't have that enabled, so I've enabled this one property instead of strict as that would likely be too much to take on in the scope of this PR. I've had to add a `@ts-ignore` for all occurrences in classes that aren't initializing all properties in the constructor. If not ignored, it would result in errors that look like this:

> TS2564: Property 'params' has no initializer and is not definitely assigned in the constructor.

This means that all the integration-related classes have these `@ts-ignore` and `eslint-disable-next-line` in them. I didn't want to assume values and potentially change existing functionality which is why I'm ignoring them instead of assigning values.